storage/card.go

```diff
@@ -1,10 +1,16 @@
-// https://github.com/usbarmory/armory-witness-boot
+// Copyright 2022 The ArmoredWitness Authors. All Rights Reserved.
```

Comment:
What does this refer to exactly?  The AUTHORS file?

Reply:
Yes, I can change to Armored Witness Boot authors if you think that's better?

Reply:
I will admit that I know very little about copyrights. I don't know if the syntactic sugar used in the .go files, "The ArmoredWitness Authors" needs to be officially defined somewhere? Is an AUTHORS file enough, or does every .go file need to say "The ArmoredWitness Authors are listed in the AUTHORS files of XX"?

Reply:
Ah, an AUTHORS file seems to be enough indeed! https://opensource.google/documentation/reference/releasing/authors. Yup, sounds good to me to rephrase this to "Armored Witness Boot" to match with the repo's name indeed.

Reply:
done